### PR TITLE
Implement Liquid support

### PR DESCRIPTION
**Pull request by stevenroose**
Does full Elements/Liquid signing, but uses the node to blind the tx.

### CODE REVIEW

File: src/coins/liq.rs [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+use crate::errors::{Error, OptionExt};
+use crate::network::ElementsNetwork;
+use crate::throw;
+use crate::util;
+use crate::wallet::AddressMeta;
+
+lazy_static! {
+    pub static ref LBTC: rpcjson::AssetId = rpcjson::AssetId::from_hex(
+        "6f0279e9ed041c3d710a9f57d0c02928416460c4b722ae3457a11eec381c526d"
```
Comment (RCasatta) [inline src/coins/liq.rs]: can't LBTC and EBTC be a &str so they can both be used for `asset()` and `asset_hex()`

File: test.sh [hunk 2]
```diff
@@ -5,11 +5,70 @@ if [ -f /.dockerenv ]; then
...
 
-bitcoin-cli -regtest -rpcwait generatetoaddress 200 $(bitcoin-cli -regtest -rpcwait getnewaddress)
+TEST_MODE=${TEST_MODE:-bitcoinregtest}
 
-BITCOIND_DIR=~/.bitcoin/regtest \
-cargo test --features stderr_logger --all  -- --test-threads=1
 
-bitcoin-cli -regtest stop
+echo "Killing existing nodes..."
+killall -9 $EXEC_NODE || true
```
Comment (RCasatta) [inline test.sh]: not a fan of this, it closed the mainnet node on my machine, why change the `bitcoin-cli -regtest stop` which target only regtest node?

Comment (stevenroose) [reply test.sh]: Hmm, I remember changing this for an edge case, probably should have changed this back. I think somehow nodes got unresponsive during startup or wallet loading time or so. Will revert and do manual kill in the occasional issue.

File: src/coins/liq.rs (hunk 1 above)
Comment (stevenroose) [reply src/coins/liq.rs]: True, optimized this.
